client/package.json

```diff
@@ -187,17 +193,20 @@
 		"react-lazily-render": "^1.2.0",
 		"react-live": "^1.12.0",
 		"react-modal": "^3.11.1",
+		"react-native": "^0.63.4",
```

Comment:
Ooof… you're actually pulling in `react-native`? That's a totally new dep isn't it? I'm pretty sure it's not required for any web usage, just leaking in from here: https://github.com/WordPress/gutenberg/blob/4e0dceaf94596d76c25abe1542378aa4d6c6d035/packages/url/package.json#L32

Reply:
I silenced this warning that seems to be bogus with the following instead of installing react-native:

```yaml
  '@wordpress/url@^3.1.1':
    peerDependenciesMeta:
      react-native:
        optional: true
```

Reply:
It has been there for more than a year: https://github.com/WordPress/gutenberg/pull/20537/files 
And it has caused problems a few times: p1624469210216700-slack-C02DQP0FP

But I don't feel comfortable altering the dependency tree that way. Right now `@wordpress/url` already has `react-native`, so any change in `@wordpress/url` can make use of it in a way that it breaks if it is not present.